AEPIdentity/Sources/IdentityState.swift

```diff
@@ -16,15 +16,16 @@ import Foundation
 /// Manages the business logic of the Identity extension
 class IdentityState {
     private let LOG_TAG = "IdentityState"
-    private(set) var hitQueue: HitQueuing
     private var pushIdManager: PushIDManageable
     #if DEBUG
+        var hitQueue: HitQueuing
```

Comment:
For functional testing can we use the network request mock instead? that will not require code changes and it will maintain the expected input/output format for functional tests.